omnibus/files/private-chef-cookbooks/private-chef/recipes/rabbitmq.rb

```diff
@@ -1,300 +1 @@
-    environment rabbitmq_env
-    user opc_username
-    not_if "#{rmq_ctl_chpst} list_user_permissions #{rabbitmq['actions_user']}|grep #{rabbitmq['actions_vhost']}", environment: rabbitmq_env, user: 'root'
-    retries 10
-  end
-
-  execute "#{rmq_ctl} set_permissions -p #{rabbitmq['actions_vhost']} #{rabbitmq['management_user']} \".*\" \".*\" \".*\"" do
-    environment rabbitmq_env
-    user opc_username
-    not_if "#{rmq_ctl_chpst} list_user_permissions #{rabbitmq['management_user']}|grep #{rabbitmq['actions_vhost']}", environment: rabbitmq_env, user: 'root'
-    retries 10
-  end
-
-  execute "#{rmq_ctl} set_permissions -p #{rabbitmq['vhost']} #{rabbitmq['management_user']} \".*\" \".*\" \".*\"" do
-    environment rabbitmq_env
-    user opc_username
-    not_if "#{rmq_ctl_chpst} list_user_permissions #{rabbitmq['management_user']}|grep #{rabbitmq['vhost']}", environment: rabbitmq_env, user: 'root'
-    retries 10
-  end
-
-  execute "#{rmq_ctl} set_permissions -p / #{rabbitmq['management_user']} \".*\" \".*\" \".*\"" do
-    environment rabbitmq_env
-    user opc_username
-    not_if "#{rmq_ctl_chpst} list_user_permissions #{rabbitmq['management_user']}|grep \"/\\s\"", environment: rabbitmq_env, user: 'root'
-    retries 10
-  end
-
-  rabbitmq_management_is_up = "#{rmq_plugins} list | grep rabbitmq_management  | grep -v rabbitmq_management_agent | grep -v rabbitmq_management_visualiser | grep E"
-  if rabbitmq['management_enabled']
-    execute "#{rmq_plugins} enable rabbitmq_management" do
-      environment rabbitmq_env
-      user opc_username
-      not_if rabbitmq_management_is_up
-      # management plugin needs a rabbit restart
-      notifies :run, 'script[hard_kill_rabbitmq]', :delayed
-      retries 10
-    end
-  else
-    execute "#{rmq_plugins} disable rabbitmq_management" do
-      environment rabbitmq_env
-      user opc_username
-      notifies :run, 'script[hard_kill_rabbitmq]', :delayed
-      only_if rabbitmq_management_is_up
-      retries 10
-    end
-  end
-
-  execute "#{rmq_ctl} set_user_tags #{rabbitmq['management_user']} administrator" do
-    environment rabbitmq_env
-    user opc_username
-    not_if "#{rmq_ctl_chpst} list_users | grep #{rabbitmq['management_user']} | grep administrator", environment: rabbitmq_env, user: 'root'
-    retries 10
-  end
-
-  execute "#{rmq_ctl} set_policy -p /analytics max_length '(erchef|alaska|notifier.notifications|notifier_config)' '{\"max-length\":#{rabbitmq['analytics_max_length']}}' --apply-to queues" do
-    environment rabbitmq_env
-    user opc_username
-    only_if { rabbitmq['analytics_max_length'] > 0 }
-    retries 10
-  end
-
-  execute "#{rmq_ctl} clear_policy -p /analytics max_length" do
-    environment rabbitmq_env
-    user opc_username
-    not_if { rabbitmq['analytics_max_length'] > 0 }
-    retries 10
-  end
-
-end
+raise "Sorry, rabbitmq is no longer used in Chef Infra Server."
```

Comment:
I _think_ that given your other changes opscode-solr4 is the only recipe that has the possibility of being run, so you can probably remove this and expander completely.

Reply:
Agreed. I will merge this and create a followup PR removing the smaller bits.